backend/app/yolov8/utils.py

```diff
@@ -0,0 +1,137 @@
+import numpy as np
+import cv2
+class_names = ['person', 'bicycle', 'car', 'motorcycle', 'airplane', 'bus', 'train', 'truck', 'boat', 'traffic light',
```

Comment:
will be stored in DB, once schema is implemented until we switch to image captioning.